deploy.sh

```diff
@@ -882,11 +903,11 @@ function setup_terraform {
   EXT_PATH=$(which terraform 2> /dev/null || true)
 
   if [[ -f $TF && $($TF version | grep 'Terraform v0') == "Terraform ${TF_LATEST}" ]]; then
-    log "Terraform latest version already installed"
+    return
```

Comment:
So these messages won't be displayed during normal setup?

Reply:
Yes, as the code will always ensure now when the script is ran everything is the latest.